addons/delivery/wizard/choose_delivery_package.py

```diff
@@ -48,7 +48,7 @@ def _onchange_packaging_weight(self):
 
     def action_put_in_pack(self):
         picking_move_lines = self.picking_id.move_line_ids
-        if not self.picking_id.picking_type_id.show_reserved:
+        if not self.picking_id.picking_type_id.show_reserved and not self.env.context.get('barcode_view'):
```

Comment:
Put this commit in another branch for 14.0

Reply:
Done. PR is here: https://github.com/odoo/odoo/pull/59737